process/process_posix.go

```diff
@@ -5,6 +5,7 @@ package process
 import (
 	"context"
 	"fmt"
+	"github.com/shirou/gopsutil/internal/common"
```

Comment:
Please follow our convention to separate stdlib and 3rd-party packages in import statements.

Reply:
Done, gofmt was surprisingly switched off :)